app/policies/grade_entry_form_policy.rb

```diff
@@ -0,0 +1,22 @@
+# Policy for Grade entry forms controller
+class GradeEntryFormPolicy < ApplicationPolicy
+  default_rule :grading?
+  alias_rule :new?, :create?, :edit?, :update?, to: :manage?
```

Comment:
In general, make `manage?` the default rule. This matches the previous behaviour, where we had `before_action :authorize_only_for_admin, except: ...`. This will ensure that any new routes that are added are by default restricted to admins and "manage" TAs.